tests/plugins/magicline/manual/lineattribute.html

```diff
@@ -0,0 +1,67 @@
+<textarea cols="80" id="editor1" name="editor1" rows="10">
```

Comment:
For indentation `tab`s should be used (check `.editorconfig`), however this whole file uses spaces, please fix this.